### ISSUE TITLE
[aesthetics] When icon not visible in collections list (collections/search/index.php), show image icon but not alt text

### ISSUE THREAD

**Issue by themerekat** [Symbiota/Symbiota#2783]
<img width="1036" height="376" alt="Image" src="https://github.com/user-attachments/assets/a04b3ae6-3e6f-414d-9171-65e552216f1d" />

The alt text makes this super ugly when the image cannot be found.
